test/e2e/tests/console_stacks.test.ts

```diff
@@ -0,0 +1,29 @@
+import config from "../config";
+import TestHarness from "../harness";
+import { runPlaywrightTest } from "../runTest";
+
+it("Test source mapping of console errors.", async () => {
+  if (config.browserName !== "firefox") {
+    return;
+  }
+  await runPlaywrightTest({
+    example: "doc_exceptions_bundle.html",
+    script: async page => {
+      const harness = new TestHarness(page);
+      await harness.start();
+
+      await page.evaluate(() => app.actions.logExceptions(true));
+      await page.evaluate(() => app.actions.filterToggled("warn"));
+      await page.click("button.console-panel-button");
+
+      await harness.waitForMessage("console.trace() ConsoleTrace");
+      await harness.waitForMessage("ConsoleWarn");
+      await harness.waitForMessage("ConsoleError");
+      await harness.waitForMessage("Assertion failed: ConsoleAssert");
+      await harness.waitForMessage("Error: UncaughtError");
+      await harness.waitForMessage("Object { number: 42 }");
+      await harness.waitForMessage("Object { number: 12 }");
+      await harness.waitForMessage("uncaught exception: [object Object]");
+    },
+  });
+});
```

Comment:
Love this new test.  What do you think about putting this test into our existing test runner? This way we can 

1. write playwright and classic tests today 
2. look into using jest as a follow up for the new style and migrate the tests over